syntax = "proto3";
package user;

service UserService {
    rpc login (LoginRequest) returns (LoginResponse) {}
    rpc register (CreateRequest) returns (CreateResponse) {}
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    int32 user_id = 1;
}

message CreateRequest {
    string username = 1;
    string password = 2;
}

message CreateResponse {
    bool status = 1;
}
